cos-fleetshard-it/cucumber/src/main/java/org/bf2/cos/fleetshard/it/cucumber/ConnectorSteps.java

```diff
@@ -279,7 +280,19 @@ public void deploy_connector() {
 
     @When("^set configmap to:")
     public void change_configmap(Map<String, String> contents) {
-        getConfigMapFilter().accept(configMap -> configMap.setData(contents));
+        var sb = new StringBuilder();
+        sb.append("|-");
```

Comment:
is this needed ? this is something for yaml representation but not sire if it matter when dealing with the kube apis as  it is just json